podcasts/UpNextViewController.swift

```diff
@@ -35,7 +39,7 @@ class UpNextViewController: UIViewController, UIGestureRecognizerDelegate {
                     self.track(.upNextMultiSelectEntered)
                 }
                 if self.showingInTab {
-                    self.upNextTable.updateContentInset(multiSelectEnabled: isMultiSelectEnabled)
+                    self.multiSelectActionBarBottomConstraint.constant = PlaybackManager.shared.currentEpisode() == nil ? 8 : Constants.Values.miniPlayerOffset + 8
```

Comment:
I'm not totally sure what the 8 values are here. Could they be added to the `Constants`?

Reply:
Done in https://github.com/Automattic/pocket-casts-ios/pull/1791/commits/3c5ca74b5fb39d08e8292d7f86c9d18e4f754ad0